packages/ckeditor5-dev-translations/lib/ckeditortranslationsplugin.js

```diff
@@ -53,7 +53,8 @@ module.exports = class CKEditorTranslationsPlugin {
 			corePackageContextsResourcePath: options.corePackageContextsResourcePath || '@ckeditor/ckeditor5-core/lang/contexts.json',
 			translationsOutputFile: options.translationsOutputFile,
 			includeCorePackageTranslations: !!options.includeCorePackageTranslations,
-			skipPluralFormFunction: !!options.skipPluralFormFunction
+			skipPluralFormFunction: !!options.skipPluralFormFunction,
+			assetNamesFilter: options.assetNamesFilter || ( name => name.endsWith( '.js' ) )
```

Comment:
I found that the new property is not included in the `CKEditorTranslationsPluginOptions` definition.

https://github.com/BreyndotEchse/ckeditor5-dev/blob/b729ab3ef1bdb8fa41a871556ee89493c4c58302/packages/ckeditor5-dev-translations/lib/ckeditortranslationsplugin.js#L108

It could be specified as [`@callback`](https://jsdoc.app/tags-callback).